px-central/values.yaml

```diff
@@ -166,33 +189,75 @@ images:
     registry: 709825985650.dkr.ecr.us-east-1.amazonaws.com
     repo: portworx/jboss
     imageName: keycloak
-    tag: 21.1.1
+    tag: 21.1.2
   keycloakLoginThemeImage:
     registry: 709825985650.dkr.ecr.us-east-1.amazonaws.com
     repo: portworx
     imageName: keycloak-login-theme
-    tag: 2.2.0
+    tag: 2.7.0
   keycloakInitContainerImage:
     registry: 709825985650.dkr.ecr.us-east-1.amazonaws.com
     repo: portworx/library
     imageName: busybox
-    tag: 1.31
+    tag: 1.35.0
   mysqlImage:
     registry: 709825985650.dkr.ecr.us-east-1.amazonaws.com
     repo: portworx/library
     imageName: mysql
-    tag: 5.7.41
+    tag: 8.0.37
+  preUpgradeHookImage:
+    registry: 709825985650.dkr.ecr.us-east-1.amazonaws.com
+    repo: portworx
+    imageName: pxcentral-onprem-pre-upgrade
+    tag: 2.7.3
+  mysqlInitImage:
+    registry: 709825985650.dkr.ecr.us-east-1.amazonaws.com
+    repo: portworx
+    imageName: busybox
+    tag: 1.35.0
 
   ## Images required for enabling px-backup
-
   pxBackupImage:
     registry: 709825985650.dkr.ecr.us-east-1.amazonaws.com
     repo: portworx
     imageName: px-backup
-    tag: 2.6.0
+    tag: 2.7.3
   mongodbImage:
     registry: 709825985650.dkr.ecr.us-east-1.amazonaws.com
     repo: portworx/bitnami
     imageName: mongodb
-    tag: 5.0.20-debian-11-r13
+    tag: 7.0.14-debian-12-r0
+
+  
+
+  ## Images required for enabling px-monitor
+  pxBackupPrometheusImage:
+    registry: 709825985650.dkr.ecr.us-east-1.amazonaws.com
+    repo: portworx
+    imageName: prometheus
+    tag: v2.53.0
+  pxBackupAlertmanagerImage:
+    registry: 709825985650.dkr.ecr.us-east-1.amazonaws.com
+    repo: portworx
+    imageName: alertmanager
+    tag: v0.26.0
+  pxBackupPrometheusOperatorImage:
+    registry: 709825985650.dkr.ecr.us-east-1.amazonaws.com
+    repo: portworx
+    imageName: prometheus-operator
+    tag: v0.75.0
+  pxBackupPrometheusConfigReloaderImage:
+    registry: 709825985650.dkr.ecr.us-east-1.amazonaws.com
+    repo: portworx
+    imageName: prometheus-config-reloader
+    tag: v0.75.0
 
+  ## Images required for intermediate Mongo upgrades
+  mongodbImageMap:
+    registry: 709825985650.dkr.ecr.us-east-1.amazonaws.com
+    repo: portworx
```

Comment:
@dbinnal-px Will the repo for this also have to be `portworx/bitnami` ?

Reply:
Yes, it was followed